.github/workflows/publish.yml

```diff
@@ -21,7 +21,7 @@ jobs:
       - name: Install Node, NPM and Yarn
         uses: actions/setup-node@v3
         with:
-          node-version: 16.16.0
+          node-version: 18.12.1
```

Comment:
Can you confirm that electron.js bundles this version as well?

Reply:
I'll have to look into electron. Their docs only says TBD, but it's supposed to be 18.x. Brb

Reply:
The docs are lagging a bit but here is the info in a PR for it:
https://github.com/electron/website/pull/356/files#diff-faf92a731b6eeee77623817cb1f4aba1720bfdeac19f4937cb158b853ef2ef14R15